docs/guides/steth-integration-guide.md

```diff
@@ -109,10 +128,9 @@ shares2mint = --------------------------------------------------------------
 
 ### Do stETH rewards compound?
 
-stETH holders receive staking rewards, but those don't compound. Actual APR diminishes slightly over time for two main reasons:
+Yes, stETH rewards do compound.
```

Comment:
Maybe just 'Yes, (w)stETH rewards compound'.

Reply:
Sorry, but it kills clarity. It's better to have one token in mind and the other as a wrapper then put a (w) everywhere. I'd rather stay as it is.